README.client.rst

```diff
@@ -1,9 +1,35 @@
-Rucio Client Lite Package
+Rucio: Client Lite Package
```

Comment:
The Package names are inconsistent
> Client: Rucio: Client Lite Package
> Main: Rucio: Data Management for science in the Big Data era
> WebUI: Rucio WebUI Package

I think this should be something consistent there. Rucio Clients / Rucio / Rucio WebUI (No package)